packages/integration-sdk-testing/src/jest.ts

```diff
@@ -608,7 +608,11 @@ function toMatchSchema<T extends Entity | ExplicitRelationship>(
   received = Array.isArray(received) ? received : [received];
 
   for (let i = 0; i < received.length; i++) {
-    const data = received[i];
+    // Removes raw data before schema validation
+    const data = {
+      ...received[i],
+    };
+    delete data._rawData;
```

Comment:
const {
   _rawData,
   ...data
} = received[i];
Let's not mutate the object.

Reply:
Oh I guess it is a copy of the object. Should be fine then.

Reply:
Yeah it would have only removed it from data since it was a shallow copy